nrf52-hal-common/src/spim.rs

```diff
@@ -118,10 +118,15 @@ impl<T> Spim<T> where T: SpimExt {
         // TODO: some targets have a maxcnt whose size is larger
         // than a u8, so this length check is overly restrictive
         // and could be lifted.
```

Comment:
I think after this commit, this `TODO` no longer applies.

Reply:
Well, it kind of does, as I did it wrong. Apparently the register is only 10 bits wide, not 16.
I will fix this as soon as I get UART and SPI working. Atm it's behaving strangely. Uart::write() completes without any errors but the pin does not actually toggle (It goes high during the initialization of the UART tho).
I have no clue what this could be but maybe there are some defs wrong. Still trying to figure svd2rust generated code out to verify :)
Thanks for the hint/reminder